aries_cloudagent/protocols/issue_credential/v2_0/models/detail/vc_di.py

```diff
@@ -0,0 +1,135 @@
+"""Indy-specific credential exchange information with non-secrets storage."""
+
+from typing import Any, Mapping, Sequence
+
+from marshmallow import EXCLUDE, fields
+
+from ......core.profile import ProfileSession
+from ......messaging.models.base_record import BaseRecord, BaseRecordSchema
+from ......messaging.valid import (
+    INDY_CRED_REV_ID_EXAMPLE,
+    INDY_CRED_REV_ID_VALIDATE,
+    INDY_REV_REG_ID_EXAMPLE,
+    INDY_REV_REG_ID_VALIDATE,
+    UUID4_EXAMPLE,
+)
+from .. import UNENCRYPTED_TAGS
+
+
+class V20CredExRecordVCDI(BaseRecord):
+    """Credential exchange vc_di detail record."""
+
+    class Meta:
+        """V20CredExRecordIndy metadata."""
+
+        schema_class = "V20CredExRecordVCDISchema"
+
+    RECORD_ID_NAME = "cred_ex_vc_di_id"
+    RECORD_TYPE = "vc_di_cred_ex_v20"
+    TAG_NAMES = {"~cred_ex_id"} if UNENCRYPTED_TAGS else {"cred_ex_id"}
+    RECORD_TOPIC = "issue_credential_v2_0_vc_di"
+
+    def __init__(
+        self,
+        cred_ex_vc_di_id: str = None,
+        *,
+        cred_ex_id: str = None,
+        cred_id_stored: str = None,
+        cred_request_metadata: Mapping = None,
+        rev_reg_id: str = None,
+        cred_rev_id: str = None,
+        **kwargs,
+    ):
+        """Initialize indy credential exchange record details."""
+        super().__init__(cred_ex_vc_di_id, **kwargs)
+
+        self.cred_ex_id = cred_ex_id
+        self.cred_id_stored = cred_id_stored
+        self.cred_request_metadata = cred_request_metadata
+        self.rev_reg_id = rev_reg_id
+        self.cred_rev_id = cred_rev_id
+
+    @property
+    def cred_ex_vc_di_id(self) -> str:
+        """Accessor for the ID associated with this exchange."""
+        return self._id
+
+    @property
+    def record_value(self) -> dict:
+        """Accessor for the JSON record value generated for this credential exchange."""
+        return {
+            prop: getattr(self, prop)
+            for prop in (
+                "cred_id_stored",
+                "cred_request_metadata",
+                "rev_reg_id",
+                "cred_rev_id",
+            )
+        }
+
+    @classmethod
+    async def query_by_cred_ex_id(
+        cls,
+        session: ProfileSession,
+        cred_ex_id: str,
+    ) -> Sequence["V20CredExRecordVCDI"]:
+        """Retrieve credential exchange indy detail record(s) by its cred ex id."""
+        return await cls.query(
+            session=session,
+            tag_filter={"cred_ex_id": cred_ex_id},
+        )
+
+    def __eq__(self, other: Any) -> bool:
+        """Comparison between records."""
+        return super().__eq__(other)
+
+
+class V20CredExRecordVCDISchema(BaseRecordSchema):
+    """Credential exchange indy detail record detail schema."""
```

Comment:
should this comment say "indy" or should it say "vc_di" ?

Reply:
Yes it should. It would be updated. Thank for catching it